common/src/main/java/com/wynntils/utils/type/RangedValue.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © Wynntils 2023.
```

Comment:
See if you can revert this.

Reply:
Only by force pushing. Do you want me to?

Reply:
Yes, I think it is okay to force-push to a PR branch to get rid of a thing like this. A bit annoying that this is the only way to fix this, but in this case I think it's fine. I assume you'll make an interactive rebase and remove all edits to this file?